README.md

```diff
@@ -20,14 +20,22 @@ specific containers to use this project.
 ### Get started
 All your Drupal files should be placed into the folder "web".
 
+Add your env variable for dev.
+```sh
+$ cd to project root
+$ export BUILD_ENV=dev
+```
+
 Run docker compose.
 ```sh
 $ docker-compose up -d
 ```
 
-Import your database or install your site.
+Build local.
+Download db to project root production-db.sql.
 ```sh
-$  docker-compose run --rm php ./../vendor/bin/drush site-install
+$ cd /docroot
+$ ../scripts/drupal-update.sh
```

Comment:
I had to use $ docker-compose run --rm php ../scripts/drupal-update.sh instead of just $ ../scripts/drupal-update.sh.